### PR TITLE
Update conversion tracking for the new Facebook Pixel

### PR DESCRIPTION
**Pull request by isaacgrant**

### CODE REVIEW

**Comment by isaacgrant** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;SPTJfVwL&#x2F;38-1d-facebook-tracking-pixels"> 1d - Facebook tracking Pixels</a>

**Comment by isaacgrant** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;O5YDE1sk&#x2F;122-upgrade-to-latest-facebook-pixel-code"> Upgrade to latest Facebook Pixel code.</a>

File: Store/StoreAnalyticsOrderTracker.php [hunk 1]
```diff
@@ -48,9 +49,30 @@ public function getCommands()
...
 	// }}}
-	// {{{ protected function getOrderCommand()
+	// {{{ public function getFacebookPixelCommands()
 
-	protected function getOrderCommand()
+	public function getFacebookPixelCommands()
+	{
+		$commands = array($this->getGoogleAnalyticsOrderCommand());
+		foreach ($this->order->items as $item) {
+			$commands[] = $this->getGoogleAnalyticsOrderItemCommand($item);
```
Comment (gauthierm) [inline Store/StoreAnalyticsOrderTracker.php]: Is it correct to add GA commands to the facebook pixel API? If so, a comment is required.

Comment (isaacgrant) [inline Store/StoreAnalyticsOrderTracker.php]: It is not. I just forgot to push my commits.

**Pull request merged**